apps/hash-graph/libs/type-fetcher/predefined_types/data_types/value_v1.json

```
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://blockprotocol.org/types/modules/graph/0.3/schema/data-type",
+  "kind": "dataType",
+  "$id": "https://blockprotocol.org/@blockprotocol/types/data-type/value/v/1",
+  "title": "Value",
+  "description": "A value that can be stored in a graph",
```

The current description is circular. Not sure what a great alternative is, here's one suggestion:
```suggestion
  "description": "A piece of data that can be used to convey information about an attribute, quality or state of something.",
```

---

The description is definitely  better than my (halfway generated) one 🙂 
[`2e7e6e2` (#5240)](https://github.com/hashintel/hash/pull/5240/commits/2e7e6e204ea746842c641048e51122c88496e419)

---

I wonder if we want to extend the description and mention that every data type (directly or indirectly) is inherited from this type or if we want to leave that out.

---

I would personally leave that out, I think we could cover it in documentation about the type system.

We don't currently talk about the type system or inheritance in the type descriptions themselves, and if we're going to do so we should do so consistently across all the descriptions. At the moment the descriptions are standalone explanations of what the thing in the world is, ignoring their place in the type system.

cc @vilkinsons in case of any view on this

---

I'd also leave it out, and place it in the surrounding docs. We can then choose if it's something we pass along to LLMs, rather than it be perpetually bundled as part of the basic description (if they're ever exposed to the 'Value' data type... possibly not even, if we can avoid it).